tools/generator/templates/player/haskell/Makefile-haskell.jinja2

```diff
@@ -0,0 +1,38 @@
+{# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+#}
+{% extends "includes/Makefile-language.jinja2" %}
+
+{% block lang %}haskell{% endblock %}
+
+{% block extra_flags %}
+GHC    ?= ghc
```

Comment:
Just remove the alignment.

![image](https://user-images.githubusercontent.com/4927883/75634914-12097d80-5c12-11ea-9223-e49724cd2edb.png)